.ci/update-beats.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Bump beats
+title: "[automation] update libbeat and beats packaging"
+
+scms:
+  apm-server:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-server
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: '{{ requiredEnv "BRANCH_NAME" }}'
+
+actions:
+  apm-server-pr:
+    kind: github/pullrequest
+    scmid: apm-server
+    spec:
+      title: "[automation] update libbeat and beats packaging"
```

Comment:
I've seen in the `updatecli` project that the title is defined in a default:

https://github.com/updatecli/updatecli/blob/a7f4b9c75e1bcde978a5a0ebd6537a0e75440069/updatecli/updatecli.d/golang.yaml#L3-L5

and PRs honoured the title: https://github.com/updatecli/updatecli/pull/1135

Reply:
Aha.. thank you for this.. I will actually try it.

Reply:
Ok.. no clue.. why it's behaving differently here.

but this is the closes I got now..

https://github.com/elastic/apm-server/pull/10179/commits/6a293c58bbc11efe28a89b9a897cb4e3a35223fe

![image](https://user-images.githubusercontent.com/16325797/217533169-8982dfa6-a82b-4272-a0d3-61236a4c6f3b.png)

Reply:
😢  Thanks for double checking it. This is not a hard req